docker/compute_worker/compute_worker.py

```diff
@@ -205,22 +206,31 @@ def __init__(self, run_args):
         self.requests_session.mount('http://', adapter)
         self.requests_session.mount('https://', adapter)
 
-    # TODO: Rename watch_file to "detailed_results_watcher" or some such
-    async def watch_file(self):
+    async def watch_detailed_results(self):
         """Watches files alongside scoring + program docker containers, currently only used
         for detailed_results.html"""
         if not self.detailed_results_url:
             return
         file_path = os.path.join(self.output_dir, 'detailed_results.html')
+        v15_file_path = None
```

Comment:
I think we don't need this extra variable? Have `file_path` be the normal default, if nothing found at this path re-assign file_path to the first HTML file found? maybe add comments about this weird behavior